sssd_test_framework/utils/tools.py

```diff
@@ -587,6 +587,16 @@ def faillock(self, args: list[Any]) -> ProcessResult:
 
         return self.host.conn.exec(["faillock", *args])
 
+    @property
+    def sshkey(self) -> SSHKeyUtils:
+        """
+        Execute ssh-keygen command.
+        :return: SSHKeyUtils object.
+        :rtype: SSHKeyUtils
+        """
+
+        return SSHKeyUtils(self.host, self.__fs)
```

Comment:
At this point, there is only single method "generate" and you do not set any internal state. So it is not necessary to create new object every time, but it's up to you.

Reply:
Let's keep it as is. I can see us extending this eventually.